### PR TITLE
Shipping Labels: Update logo for DHL

### LINKED ISSUE #4888: Shipping Label creation: the image of FedEx courier does not respect the aspect ratio in Carrier and Rates
I found a small bug, where the image of FedEx courier does not respect the aspect ratio of the image in the Carrier and Rates screen in the Shipping Label creation flow.

### PR DESCRIPTION
**Pull request by itsmeichigo**
Fixes #4888 

# Description
This PR updates logo for DHL (previously FedEx's). The logo is square so the aspect ratio is good too.

# Screenshots
| Before | After |
| ----- | ----- |
|  | 

# Testing
1. Make sure your test store has installed and activated WCShip plugin.
2. Select an international order (value around $200 to make sure DHL is available 🤞) > Create Shipping Label.
3. Configure Ship From, Ship To, Package Details, Customs form.
4. On Carriers and Rates screen: notice that the logo for DHL is correct and aspect ratio is good too.

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 - [*] Fix: now a default paper size will be selected in Shipping Label print screen. [https://github.com/woocommerce/woocommerce-ios/pull/5035]
 - [*] Show banner on screens that use cached data when device is offline. [https://github.com/woocommerce/woocommerce-ios/pull/5000]
 - [*] Fix incorrect subtitle on customs row of Shipping Label purchase flow. [https://github.com/woocommerce/woocommerce-ios/pull/5093]
+- [*] Fix incorrect logo for DHL. [https://github.com/woocommerce/woocommerce-ios/pull/5105]
```
Comment (pmusolino) [inline RELEASE-NOTES.txt]:
```suggestion
- [*] Fix incorrect logo for DHL in Shipping Labels. [https://github.com/woocommerce/woocommerce-ios/pull/5105]
```

**Review by pmusolino — CHANGES_REQUESTED** [review]
LGTM. 👍 

Just a nit on release notes and a question.
Q: Should we remove the image `shipping-label-fedex-logo` since it's no more used? I think it's worth removing it for now.

**Comment by pmusolino** [comment]
Can I take a second look @itsmeichigo? I saw that you submitted the changes, but did not request a new review. 😃

**Comment by itsmeichigo** [comment]
Oops sorry @pmusolino! I thought I got a green light (so many PRs today 😅). Just requested your review again!

**Review by pmusolino — APPROVED** [review]
🚀

**Pull request merged**